rcl_yaml_param_parser/src/parser.c

```diff
@@ -1366,45 +1363,41 @@ bool rcl_parse_yaml_file(
   const char * file_path,
   rcl_params_t * params_st)
 {
-  int32_t res;
-  FILE * yaml_file;
-  yaml_parser_t parser;
-  namespace_tracker_t ns_tracker;
-  rcutils_allocator_t allocator;
-
   if (NULL == params_st) {
     RCUTILS_SAFE_FWRITE_TO_STDERR("Pass a initialized paramter structure");
     return false;
   }
-  allocator = params_st->allocator;
+  rcutils_allocator_t allocator = params_st->allocator;
 
   if (NULL == file_path) {
-    RCL_SET_ERROR_MSG("YAML file path is NULL");
+    RCUTILS_SET_ERROR_MSG("YAML file path is NULL");
     return false;
   }
```

Comment:
Use `RCUTILS_CHECK_ARGUMENT_FOR_NULL`

Reply:
Done in cf8bff1e0420edf110631ff124963d6a199f638f.